src/main/groovy/com/vanniktech/android/junit/jacoco/GenerationPlugin.groovy

```diff
@@ -63,8 +63,9 @@ class GenerationPlugin implements Plugin<Project> {
             }
 
             classDirectories = subProject.fileTree(
-                    dir: 'build/classes/', // Starting from Gradle 4.+ each language has it's own classes folder like build/classes/java/main or build/classes/java/test.
-                    excludes: getExcludes(extension) + '**/test/**' // We'll exclude the test directory.
+                    dir: subProject.buildDir,
+                    includes: ["**/classes/**"], // Starting from Gradle 4.+ each language has it's own classes folder like build/classes/java/main or build/classes/java/test.
```

Comment:
remove the comment as this should just work now.

Reply:
Done